### PR TITLE
Translate quick-start documentation to English.

### PR DESCRIPTION
**Pull request by CH3CHO**
Shall I commit the build results as well?

### CODE REVIEW

**Comment by CH3CHO** [comment]
https://github.com/alibaba/higress/issues/68

**Comment by johnlanni** [comment]
> Shall I commit the build results as well?

The .md file is enough.

File: docs/en-us/user/quickstart.md [hunk 1]
```diff
@@ -6,4 +6,147 @@ description: Quick Start.
...
+kubectl create ns istio-system
+helm install istio -n istio-system oci://higress-registry.cn-hangzhou.cr.aliyuncs.com/charts/istio-local
+kubectl create ns higress-system
+helm install higress -n higress-system oci://higress-registry.cn-hangzhou.cr.aliyuncs.com/charts/higress-local
+```
+
+### Step 4: Create Ingress Resource and Test
+
+```bash
+kubectl apply -f https://kind.sigs.k8s.io/examples/ingress/usage.yaml
```
Comment (johnlanni) [inline docs/en-us/user/quickstart.md:142]:
```suggestion
kubectl apply -f https://github.com/alibaba/higress/releases/download/v0.5.2/quickstart.yaml
```

这个配置换成了国内镜像，可以方便用户拉取

File: docs/en-us/user/quickstart.md [hunk 2]
```diff
@@ -6,4 +6,147 @@ description: Quick Start.
...
+```
+
+Test Ingress routes:
+
+```bash
+# should output "foo-app"
+curl localhost/foo/hostname
```
Comment (johnlanni) [inline docs/en-us/user/quickstart.md:149]: 这里可以复原一下，用上面这份配置，不会更新了

File: docs/zh-cn/user/quickstart.md [hunk 3]
```diff
@@ -145,8 +145,8 @@ kubectl apply -f https://kind.sigs.k8s.io/examples/ingress/usage.yaml
 测试 Ingress 生效：
 
 ```bash
-# should output "foo"
-curl localhost/foo
-# should output "bar"
-curl localhost/bar
+# should output "foo-app"
```
Comment (johnlanni) [inline docs/zh-cn/user/quickstart.md:148]: 可以复原一下

**Review by johnlanni — APPROVED** [review]

File: docs/en-us/user/quickstart.md (hunk 2 above)
Comment (CH3CHO) [reply docs/en-us/user/quickstart.md]: Done

File: docs/zh-cn/user/quickstart.md (hunk 3 above)
Comment (CH3CHO) [reply docs/zh-cn/user/quickstart.md]: Done

**Pull request merged**